### ISSUE TITLE
Can I add image in output patterns tab?

### ISSUE THREAD

**Issue by VectrorX** [Piotrekol/StreamCompanion#287]
I added image below pp counter to obs but when I'm not playing map, it doesn't hide like counter does. Is there anyway that i can insert image into Formatting in Output patterns, so I can just save event as playing and it will hide with the pp counter?

Sorry for English if there's something wrong, hard question to translate for me.

**Comment by Piotrekol** [comment]
From what I understood you want to show or hide map background image depending on the current osu! status.
To do that you'll need to create a custom web overlay.

If that's really the only thing you need, you can try asking on discord(linked in readme) in #sc-overlays.

**Comment by Piotrekol** [comment]
explained how to do conditional showing/hiding in SC overlays via discord.

tl;dr:
```
let isPlayingOrWatching = Vue.computed(() =>
      _IsInStatus(data.rws, data.tokens, [window.overlay.osuStatus.Playing, window.overlay.osuStatus.Watching])
    );
```
![image](https://user-images.githubusercontent.com/4990365/115204251-91992b00-a0f8-11eb-9c3a-6ec23d41ad63.png)
![image](https://user-images.githubusercontent.com/4990365/115204289-9b229300-a0f8-11eb-87ee-9bb132776013.png)
